src/NuGet.Core/NuGet.Protocol/Resources/RawSearchResourceV3.cs

```diff
@@ -36,6 +38,7 @@ public RawSearchResourceV3(HttpSource client, IEnumerable<Uri> searchEndpoints)
             _searchEndpoints = searchEndpoints.ToArray();
         }
 
+        [Obsolete]
```

Comment:
```suggestion
        [Obsolete("Use PackageSearchResource instead (via SourceRepository.GetResourceAsync<PackageSearchResource>")]
```

It's really not obvious to anyone using this API what they should be using instead.

Reply:
Ok. Done.